stl/inc/type_traits

```diff
@@ -1473,6 +1473,14 @@ _NODISCARD constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_c
     return _STD move(_Arg);
 }
 
+// std::invoke isn't constexpr in C++17, and implementers are forbidden from "strengthening"
+// constexpr, yet both std::apply and std::visit are required to be constexpr and have invoke-like behavior. We solve
+// this by using a macro to stamp out both public non-constexpr and internal constexpr implementations.
+
+// Now that a Core Language issue affecting constexpr invoke() has been fixed (CWG1581), we may make this unconditional

```

Comment:
Should hyphenate `CWG-1581` for consistency with LWG issues (although we don't have CWG autolinks, and autolinks aren't active in source code).